accounts/accounts/settings.py

```diff
@@ -19,7 +19,7 @@
 #USER_AGENT = 'accounts (+http://www.yourdomain.com)'
 
 # Obey robots.txt rules
-ROBOTSTXT_OBEY = True
+ROBOTSTXT_OBEY = False
```

Comment:
It does not change the result of the crawler, just change it because too many logs in the output, less requests now